### DISCUSSION TITLE
How to add a remote Sensor to Tasmota from Tasmota Zigbee Bridge

### DISCUSSION THREAD

**Discussion by jonij** [arendst/Tasmota#21096] [Q&A]
I want to add several Zigbee devices to HomeKit via matter. I have a Sonoff ZbBridge flashed with Tasmota and have that connected to another Tasmota device (ESP32-C3) which functions as matter bridge.

Zigbee sensor -> ZbBridge -> Tasmota (matter bridge) -> HomeKit

<img width="379" alt="Bildschirmfoto 2024-04-04 um 18 23 49" src="https://github.com/arendst/Tasmota/assets/5861989/55c8f6c3-c959-4fde-b13e-96da98e10c3d">
<img width="368" alt="Bildschirmfoto 2024-04-04 um 18 23 28" src="https://github.com/arendst/Tasmota/assets/5861989/a84eb7f0-ef0f-433a-a75a-c477a708bbd0">

I don't know what Parameter I have to use to connect the Zigbee devices from the remote ZbBridge

**Comment by s-hadinger** [comment]
Zigbee to Matter integration is not done. It will come in the future. For now you can somehow emulate this with well crafted rules

**Reply by jonij** [reply]
Thanks, I will try to do it with rules
